app/src/main/java/com/architectcoders/arquitectomarvel/data/remote/models_moshi/comics/Result.kt

```diff
@@ -0,0 +1,99 @@
+package com.architectcoders.arquitectomarvel.data.remote.models_moshi.comics
+
+import android.os.Parcelable
+import com.architectcoders.arquitectomarvel.data.remote.models_moshi.characters.Thumbnail
+import com.architectcoders.arquitectomarvel.data.remote.models_moshi.characters.fromComicsThumbailDomain
+import com.architectcoders.arquitectomarvel.data.remote.models_moshi.characters.toComicsThumbailDomain
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+import kotlinx.parcelize.Parcelize
+import com.architectcoders.module.domain.remote_models.Comics.Result as ComicsResultDomain
+
+@Parcelize
+@JsonClass(generateAdapter = true)
+data class Result(
+//    @Json(name = "characters")
```

Comment:
@joseantoniocampillo could you remove the commented code?

Reply:
We are making a call that can receive all of that. I see that it is the perfect place (our moshi object), where in the future we can "open the door" to a new parameter just by uncommenting. And we have in view everything that is available.

Reply:
@joseantoniocampillo but we need to follow the YAGNI principle